scripts/build/Dockerfile.amd-rocm

```diff
@@ -0,0 +1,97 @@
+FROM rocm/pytorch:rocm4.0.1_ubuntu18.04_py3.6_pytorch
```

Comment:
This image is very large: ~5.94 GB. If we want to run tests in CI it would have a significant amount bandwidth and download time. There is a [newer version](https://hub.docker.com/r/rocm/pytorch/tags?page=1&ordering=last_updated) of the image, but it is even larger.

Is there something else that we could use for testing at CRIU level?